iteratees/src/main/scala/play/api/libs/iteratee/Iteratee.scala

```diff
@@ -671,21 +666,17 @@ trait Iteratee[E, +A] {
         case Step.Error(msg, e) => Error(msg, Input.Empty)
       }(dec)
     }(dec)
-  }
 
-  def joinConcatI[AIn, X](implicit in: A <:< Iteratee[E, AIn], p: E => scala.collection.TraversableLike[X, E], bf: scala.collection.generic.CanBuildFrom[E, X, E]): Iteratee[E, AIn] = {
-    this.flatMapTraversable { a =>
-      val inner = in(a)
-      inner.pureFlatFold[E, AIn] {
+  def joinConcatI[AIn, X](implicit in: A <:< Iteratee[E, AIn], p: E => scala.collection.TraversableLike[X, E], bf: scala.collection.generic.CanBuildFrom[E, X, E]): Iteratee[E, AIn] = this.flatMapTraversable {
```

Comment:
I think the previous formatting was clearer. I tend to agree with the Scala style guide when it comes to [the use of braces for method bodies](http://docs.scala-lang.org/style/declarations.html#body):
> When the body of a method cannot be concisely expressed in a single line or is of a non-functional nature (some mutable state, local or otherwise), the body must be enclosed in braces